sdcm/cluster_k8s/__init__.py

```diff
@@ -1764,6 +1764,25 @@ def __init__(self, name: str, parent_cluster: PodCluster, node_prefix: str = "no
             dc_idx=dc_idx,
             rack=rack)
         self.k8s_cluster = self.parent_cluster.k8s_clusters[self.dc_idx]
+        self._rack_name = None
+
+    # @cached_property
+    # def datacenter(self) -> str:
+    #     return self.k8s_cluster.region_name
+
+    @property
+    def node_rack(self) -> str:
+        if not self._rack_name:
+            self._rack_name = self._pod.get("metadata", {}).get(
+                "labels", {}).get("scylla/rack", "") if self._pod else ""
```

Comment:
According to the ci job error the return object is not dict.
So, need to update it to be the following:

```
self._rack_name = self._pod.metadata.labels.get("scylla/rack", "") if self._pod else ""
```

Reply:
fixed